doc/readthedocs/geometrydefinition.rst

```diff
@@ -181,16 +180,23 @@ For SPH, the following different phases may be defined:
 - neumannphase
 - dirichletphase
 
-The radius of the particles are by default defined in the material definition. For SPH, this is the only option available.
-For DEM, however, as mentioned above, they can also be defined per particle.
-For this definition option, one has to define the option :ref:`INITIAL_RADIUS <particledynamic_dem_initial_radius>` in :ref:`PARTICLE DYNAMIC <SECparticledynamic_dem>`::
+Note that with SPH the particles are allowed to change its phase during a simulation based on the input parameter `PHASECHANGETYPE` and `PHASECHANGEDEFINITION` in the section :ref:`PARTICLE DYNAMIC/SPH <SECparticledynamic_sph>`.
```

Comment:
Maybe @ischeider can confirm this: I think we should use double backticks, right?

```suggestion
Note that with SPH the particles are allowed to change its phase during a simulation based on the input parameter ``PHASECHANGETYPE`` and ``PHASECHANGEDEFINITION`` in the section :ref:`PARTICLE DYNAMIC/SPH <SECparticledynamic_sph>`.
```

Reply:
You are right, it should be double backticks.